lib/ecto/queryable.ex

```diff
@@ -42,8 +42,19 @@ end
 
 defimpl Ecto.Queryable, for: Tuple do
   def to_query({source, %Ecto.Query{from: {_, schema}} = query}) when is_binary(source),
```

Comment:
I actually did not even rmember this syntax being allowed. I will investigate. :D